packages/next/src/build/turbopack-build/impl.ts

```diff
@@ -140,7 +140,7 @@ export async function turbopackBuild(): Promise<{
       )
     }
 
-    if (topLevelErrors.length > 0) {
+    if (topLevelErrors.length > 0 || !('routes' in entrypoints)) {
```

Comment:
this would create a confusing error message if it happens `Turbopack build failed with 0 errors:`

break out a separate message?